src/components/timeline/TrackMemoryGraph.js

```diff
@@ -463,7 +465,7 @@ class TrackMemoryGraphImpl extends React.PureComponent<Props, State> {
     }
     const { minCount, countRange, accumulatedCounts } = accumulatedSamples[0];
     const unitSampleCount =
-      (accumulatedCounts[counterIndex] - minCount) / countRange;
+      (accumulatedCounts[counterIndex] - minCount) / Math.max(1, countRange);
```

Comment:
can you please explain why this change is necessary?

Reply:
if the all counter number are 0, then this causes a division by zero without the fix